### PR TITLE
Add n88util

### PR DESCRIPTION
**Pull request by Besler**
These are some utility functions for the open source numerics88 software suite. See https://github.com/Numerics88. The objective is to distribute all four repositories (n88util, AimIO, n88tools, and vtkbone) on conda thanks to the conda-forge community.

### CODE REVIEW

**Comment by Besler** [comment]
@conda-forge/staged-recipes Is it possible to get a quick review? I apologize if I'm stepping out of line.

File: recipes/n88util/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	-G "Ninja" \
+  -DCMAKE_BUILD_TYPE=$BUILD_CONFIG \
+  -DCMAKE_PREFIX_PATH:PATH="${PREFIX}" \
+  -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+	-DCMAKE_INSTALL_RPATH:PATH="${PREFIX}/lib" \
+	-DENABLE_TESTING:BOOL=OFF \
+	-DBUILD_SHARED_LIBS:BOOL=ON
+
+# Compile and install!
+ninja install
```
Comment (bgruening) [inline recipes/n88util/build.sh]: Is there something like `make test` which we can run here?

File: recipes/n88util/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+# Create build directory
+mkdir build
+cd build
+BUILD_CONFIG=Release
+
+# CMake
+cmake .. \
+	-G "Ninja" \
+  -DCMAKE_BUILD_TYPE=$BUILD_CONFIG \
```
Comment (bgruening) [inline recipes/n88util/build.sh]: indentation ...

File: recipes/n88util/bld.bat [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+:: Create build directory
+mkdir build
+cd build
+set BUILD_CONFIG=Release
+
+:: CMake
+cmake .. ^
+	-G "Ninja" ^
+  -DCMAKE_BUILD_TYPE=%BUILD_CONFIG% ^
```
Comment (bgruening) [inline recipes/n88util/bld.bat]: indentation

Comment (Besler) [reply recipes/n88util/bld.bat]: Thank you! I had spaces in some places, tabs in others. Very dumb.

File: recipes/n88util/build.sh (hunk 2 above)
Comment (Besler) [reply recipes/n88util/build.sh]: Again, spaces vs tabs. Sorry about that.

File: recipes/n88util/build.sh (hunk 1 above)
Comment (Besler) [reply recipes/n88util/build.sh]: Yes! I added `ctest -V` and set `ENABLE_TESTING` on.

**Pull request closed without merging**